app/src/main/res/values/strings.xml

```diff
@@ -440,9 +440,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="ends_on">Ends on:</string>
   <string name="display_campaigns">Display campaigns</string>
   <string name="display_campaigns_explanation">See the ongoing campaigns</string>
-  <string name="get_content_photo_picker_title">Use GET_CONTENT photo picker</string>
-  <string name="get_content_photo_picker_explanation">Disable if your pictures get uploaded without location</string>
-  <string name="location_loss_warning">Please make sure that this new Android picker does not strip location from your pictures.</string>
+  <string name="open_document_photo_picker_title">Use document based photo picker</string>
+  <string name="open_document_photo_picker_explanation">The new Android photo picker has the potential to strip location information. Enable if you seem to be using it.</string>
+  <string name="location_loss_warning">Turning this off could trigger the new Android photo picker. It has potential to strip location information.\n\nTap on \'Read more\' for more information.</string>
```

Comment:
If my understanding is correct, toggling the switch would definitely trigger the new Android photo picker right?

If yes, we should modify - "could" to "would" in the following line - Turning this off "could" trigger the new Android photo picker.

Reply:
> toggling the switch would definitely trigger the new Android photo picker right?

Not always. It will be triggered only when the `GET_CONTENT` takeover is enabled on a device (and as of now, we do not know how to detect its status).

Reply:
Oh then it makes sense!